app/controllers/users_controller.rb

```diff
@@ -22,7 +26,12 @@ def edit
   end
 
   def update
-
+    @user.update_attributes(user_params)
+    if @user.save
```

Comment:
update_attributes has already done the save for you. I'm not sure this'll work. You might want assign_attributes which does the assignment but not the save.